support-frontend/assets/pages/showcase/showcase.jsx

```diff
@@ -27,16 +30,35 @@ import ConsentBanner from 'components/consentBanner/consentBanner';
 import './showcase.scss';
 import { Provider } from 'react-redux';
 
+// ----- Internationalisation ----- //
+
+const countryGroupId: CountryGroupId = detect();
+
+const CountrySwitcherHeader = headerWithCountrySwitcherContainer({
+  path: '/support',
+  countryGroupId,
+  listOfCountries: [
```

Comment:
`listOfCountryGroups` might be more accurate?

Reply:
I have updated this - it has knock-on effects to all the other pages that have a header with the country switcher, so I've updated all those as well.